veggieseasons/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -181,6 +182,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
+						DevelopmentTeam = S8QB4VV633;
```

Comment:
I wouldn't check this in (since it's in samples and people should be able to clone and build themselves)